### ISSUE TITLE
Alt text duplicates existing text

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4107]
## Area or Component
Static site - getting started page

## Quick Summary
The search bar contains an `<img>` element with the alt text "Search" that duplicates existing text.

### How to Replicate
1. Open https://www.simplereport.gov/getting-started/
2. Inspect the search bar and see that the `<button>` element contains both a hidden `<span>` with text "Search" and an `<img>` with alt text of "Search"

## Accessibility Violation
Alternative text of images should not be repeated as text: https://dequeuniversity.com/rules/axe/4.4/image-redundant-alt?application=AxeChrome

## Action Requested
Remove the redundant `<span>` element.

## Screenshots (optional)
![Screen Shot 2022-07-28 at 2 31 21 PM](https://user-images.githubusercontent.com/80282552/181641188-1789f269-d8dc-4569-afd6-cc7a76c878e6.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1510420308&range=11:11
